drom.opam

```diff
@@ -26,7 +26,7 @@ bug-reports: "https://github.com/ocamlpro/drom/issues"
 dev-repo: "git+https://github.com/ocamlpro/drom.git"
 tags: "org:ocamlpro"
 build: [
-  ["dune" "subst"] {dev}
```

Comment:
In this comment https://github.com/ocaml/opam-repository/pull/17730#issuecomment-735456736 , Kate asked us to switch from `{pinned}` to `{dev}`

Reply:
This file is generated, I didn't touched it by hand, so I'm not sure why it changed.